### PR TITLE
removed scroll bar for large screens and customised it in the why cla…

### LINKED ISSUE #125: Remove Scroll Bar for larger screens and beautify it for smaller ones
image
The scroll bar can be removed for larger screens. Also, it can be beautified for the smaller ones according to the theme of the website. If it seems a good idea, can I work on it?

### PR DESCRIPTION
**Pull request by developer-diganta**
…ss quiz

#### :tophat: What? Why?
Removed the scroll bar from why class quiz section for large screens and styled it for smaller ones. But had to use custom CSS for styling.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #125 ?
- Fixes #125 ?

#### Testing
*Describe the best way to test or validate your PR.*
Open webpage for small and large screens and navigate to why class quiz section.

#### :clipboard: Checklist
⚠️  No tests suites for now ⚠️
:rotating_light: Please review the [guidelines for contributing](https://github.com/mawoka-myblock/ClassQuiz/blob/master/CONTRIBUTING.md) to this repository.

- [ ] :question: ~~**CONSIDER** adding a unit test if your PR resolves an issue.~~
- [x] :heavy_check_mark: **DO** check open PR's to avoid duplicates.
- [x] :heavy_check_mark: **DO** keep pull requests small so they can be easily reviewed.
- [x] :heavy_check_mark: **DO** build locally before pushing.
- [x] :heavy_check_mark: ~~**DO** make sure tests pass.~~
- [x] :heavy_check_mark: ~~**DO** add CHANGELOG upgrade notes if required.~~
- [x] :x:~~**AVOID** breaking the continuous integration build.~~
- [x] :x:**AVOID** making significant changes to the overall architecture.

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
image

:hearts: Thank you!

### CODE REVIEW

File: frontend/src/app.css [hunk 1]
```diff
@@ -32,3 +32,24 @@
...
+#why-classquiz::-webkit-scrollbar-thumb {
+	box-shadow: inset 0 0 10px 10px #374151;
+	border: solid 3px transparent;
+	border-radius: 15px;
+}
+
+#why-classquiz::-webkit-scrollbar-thumb:hover {
+	box-shadow: inset 0 0 10px 10px #555;
+	border: solid 3px transparent;
+}
```
Comment (mawoka-myblock) [inline frontend/src/app.css:55]: Please move the CSS from the `app.css`-file into the `frontend/src/routes/+page.svelte` into a `<style>`-block.

File: frontend/src/routes/+page.svelte [hunk 2]
```diff
@@ -490,7 +490,8 @@
 				</div>
 			</div>
 			<div
-				class="lg:border-l lg:border-l-black lg:border-t-0 border-t border-t-black flex lg:flex-col flex-row stretch overflow-x-scroll"
+				id="why-classquiz"
```
Comment (mawoka-myblock) [inline frontend/src/routes/+page.svelte:493]: I'd prefer a CSS-class instead of an ID.

**Review by mawoka-myblock — CHANGES_REQUESTED** [review]

File: frontend/src/routes/+page.svelte (hunk 2 above)
Comment (developer-diganta) [reply frontend/src/routes/+page.svelte:493]: Sure! I'll make the changes

**Review by mawoka-myblock — APPROVED** [review]

**Comment by mawoka-myblock** [comment]
Thank you for your work! At first, I've though that your CSS didn't change anything untill I've noticed that `-webkit-scrollbar` isn't supported by Firefox. [Source](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar#browser_compatibility)

**Pull request merged**